scripts/endpoint-diff.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+URL_1="${1:-https://mainnet.aeternity.io/mdw/}"
+URL_2="${2:-http://localhost:4000/}"
+
+ERROR=0
+SUCCESS=0
+
+echo "Diffing $URL_1 and $URL_2 ..."
+
+while read LINE; do
+  if [[ $LINE =~ mdw\/(.*?)\" ]]; then
+    ROUTE="${BASH_REMATCH[1]}"
+
+    echo "Diffing '$ROUTE'"
+
+    curl -so '.endpoint-diff.json1.tmp' "$URL_1$ROUTE"
+    curl -so '.endpoint-diff.json2.tmp' "$URL_2$ROUTE"
+
+    if diff .endpoint-diff.json1.tmp .endpoint-diff.json2.tmp; then
```

Comment:
Please redirect the output of the diff to some file under /tmp. I will start using it too. Thanks

Reply:
then we should also adjust it in the check endpoints script

Reply:
I've redirected the output of this script to a file. I think it's better to display it on the stdout though so you can see what's failing while the script is running, though

Reply:
Thanks